src/query/storages/system/src/tables_table.rs

```diff
@@ -247,8 +249,21 @@ where TablesTable<T, U>: HistoryAware
         let mut tables_ids: Vec<u64> = Vec::new();
         let mut db_name: Vec<String> = Vec::new();
 
+        let mut get_stats = true;
+        let stats_fields_indexes = [12, 13, 14, 15, 16, 17];
```

Comment:
This seems not safe?

Reply:
> let stats_fields_indexes = [12, 13, 14, 15, 16, 17];

this index is hard code by table schema If table schema modify the logic test will failed.

Reply:
better to use field names to check, column index may change in the future.